test/mysql-cdc-old-syntax/mzcompose.py

```diff
@@ -281,3 +298,90 @@ def do_inserts(c: Composition):
             """
         ),
     )
+
+
+def workflow_migration(c: Composition, parser: WorkflowArgumentParser) -> None:
+    parser.add_argument(
+        "filter",
+        nargs="*",
+        default=["*.td"],
+        help="limit to only the files matching filter",
+    )
+    args = parser.parse_args()
+
+    matching_files = []
+    for filter in args.filter:
+        matching_files.extend(glob.glob(filter, root_dir="test/mysql-cdc-old-syntax"))
+
+    # TODO: database-issues#8706
+    matching_files = [file for file in matching_files if file != "alter-source.td"]
+
+    sharded_files: list[str] = sorted(
+        buildkite.shard_list(matching_files, lambda file: file)
+    )
+    print(f"Files: {sharded_files}")
+
+    mysql_version = get_targeted_mysql_version(parser)
+
+    for file in sharded_files:
+
+        mz_old = Materialized(
+            name="materialized",
+            image=get_old_image_for_source_table_migration_test(),
+            external_metadata_store=True,
+            external_minio=True,
+            additional_system_parameter_defaults={
+                "log_filter": "mz_storage::source::mysql=trace,info"
+            },
+        )
+
+        mz_new = Materialized(
+            name="materialized",
+            image=get_new_image_for_source_table_migration_test(),
+            external_metadata_store=True,
+            external_minio=True,
+            additional_system_parameter_defaults={
+                "log_filter": "mz_storage::source::mysql=trace,info"
```

Comment:
@nrainer-materialize I think the issue in https://github.com/MaterializeInc/database-issues/issues/8706 is because you're not activating the migration here -- this needs to apply the `force_source_table_syntax` flag

Reply:
Oh, yes, you are right!